backend/createStaffAndRooms.py

```diff
@@ -44,45 +46,83 @@ def search_ticket(ticket, guest_entries):
     return False
 
 
-def create_rooms_main(rooms_file ="", is_hardrock=False):
+def create_rooms_main(rooms_file, is_hardrock=False):
     rooms=[]
     rooms_rows = []
     with open(rooms_file, "r") as rfile:
         for row in DictReader(rfile):
             stripd = {k.lstrip().rstrip(): v.lstrip().rstrip() for k, v in row.items() if type(k)==str and type(v)==str}
             rooms_rows.append(stripd)
 
-         
+    if(is_hardrock):
+        hotel = "Hard Rock"
+    else:
+        hotel = "Ballys"
+
+    logger.debug("read in %s rooms for %s", len(rooms_rows), hotel)
+
     for elem in rooms_rows:
-        if(elem["Placed By"]=="Roombaht"):
-            if(is_hardrock):
-                hotel = "Hard Rock"
-            else:
-                hotel = "Ballys"
-            rooms.append(Room(name_take3=elem['Room Type'],
-                              name_hotel=hotel,
-                              number=elem['Room'],
-                              available=True
-                              )
-                        )
+
+        a_room = Room(name_take3=elem['Room Type'],
+                      name_hotel=hotel,
+                      number=elem['Room']
+                      )
+
+        features = elem['Room Features (Accesbility, Lakeview, Smoking)(not visible externally)'].lower()
+        if 'hearing accessible' in features:
+            a_room.is_hearing_accessible = True
+
+        if 'ada' in features:
+            a_room.is_ada = True
+
+        if 'lakeview' in features:
+            a_room.is_lakeview = True
+
+        if 'smoking' in features:
+            a_room.is_smoking = True
+
+        if elem['Placed By'] == 'Roombaht':
+            a_room.is_available = True
```

Comment:
Maybe when the guest list is loaded and room assigned we set is_available false? Can think about other conditions under which we would want set false.

Reply:
That makes sense. Once it has a user in it (and unless the user is removed) it's no longer available. And rooms are not swappable if the `Changeable` column is set to `No`.